easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2022.05-foss-2022.05.eb

```diff
@@ -66,18 +66,14 @@ exts_list = [
         'checksums': ['1764a7f4ad58c558723c542847eb367ab0bbb6d880a4e5d5eef30a0ece5cecea'],
     }),
     ('pandas', '1.4.2', {
-        'preinstallopts': """sed -i 's@extra_compile_args = \\["-Werror"\\]@extra_compile_args = []@g' setup.py && """,
+        'preinstallopts': """sed -i 's@extra_compile_args = \["-Werror"\]@extra_compile_args = []@g' setup.py && """,
```

Comment:
I suspect you based this on an older PR that didn't have these incorrect escape sequences fixed? (here and in several more places below)

Reply:
then my second guess is that inject checksum also did these additional incorrect changes

Reply:
I have added back the changes/comments that were removed by `--inject-checksums` in https://github.com/easybuilders/easybuild-easyconfigs/pull/17413/commits/2a009c247d41a74a873dcc0d0b80d2f59d56d47c